pkg/kubelet/eviction/helpers.go

```diff
@@ -364,6 +366,16 @@ func parseThresholdStatements(statements map[string]string) ([]evictionapi.Thres
 			results = append(results, *result)
 		}
 	}
+	if utilfeature.DefaultFeatureGate.Enabled(features.InheritDefaultEvictionValues) {
+		for signal, val := range DefaultEvictionHard {
```

Comment:
if a user explicitly does not want to evict on one of the resources in `DefaultEvictionHard`, how do they accomplish that?